frontend/plan/components/search/SearchBar.tsx

```diff
@@ -0,0 +1,450 @@
+// import React, { useEffect, useState } from "react";
```

Comment:
i would delete this file in this PR if it's 100% commented out and you've got another PR open